pkg/cmd/list.go

```diff
@@ -395,6 +397,7 @@ func (w *whoCan) GetClusterRolesFor(action Action) (clusterRoles, error) {
 
 // GetRoleBindings returns the RoleBindings that refer to the given set of Role names or ClusterRole names.
 func (w *whoCan) GetRoleBindings(roleNames roles, clusterRoleNames clusterRoles) (roleBindings []rbac.RoleBinding, err error) {
+	// TODO I'm wondering if GetRoleBindings should be invoked at all when the --all-namespaces flag is specified?
```

Comment:
Interesting! I think this comes down to the semantics of `--all-namespaces`, which I now think is ambiguous: 

* If it means "who can verb resource in **any** namespace" then yes, I think we should be looking at all the possible rolebindings. 
* If it means "who can verb resource in **all** namespace" then we *could* limit to just clusterRoles, but then we'd be missing the possibility of someone who has this ability, not through a clusterRole but by being bound to a Role for every namespace

The first of these options seems more useful to me anyway. Either way, we should clarify it in the usage.

Reply:
So maybe we should have a dedicated flags `--any-namespace` and `--all-namespaces` to make it less ambiguous? Plus what you mentioned a better usage docs.